_posts/2025-04-006-not-phoeniix-aseprite-api.md

```diff
@@ -0,0 +1,32 @@
+# Improving image documentation for the Aseprite API
+For my contribution I selected the community of Aseprite. Aseprite is an open source pixel art editor that I use on a daily basis. I picked this project since I've been using it as an art program for years and I'm very familiar with it's functions and community.
```

Comment:
this blog is missing the jekyll front matter at the start that causes it to be picked up as a post (see the source code for the example blog post) - either that or the double zero in the date in the filepath may be throwing it off